docs/reference/commandline/run.md

```diff
@@ -10,109 +10,119 @@ parent = "smn_cli"
+      --label-file value            Read in a line delimited file of labels (default [])
+      --link value                  Add link to another container (default [])
+      --link-local-ip value         Container IPv4/IPv6 link-local addresses (default [])
+      --log-driver string           Logging driver for container
+      --log-opt value               Log driver options (default [])
+      --mac-address string          Container MAC address (e.g. 92:d0:c6:0a:29:33)
+  -m, --memory string               Memory limit
+      --memory-reservation string   Memory soft limit
+      --memory-swap string          Swap limit equal to memory plus swap: '-1' to enable unlimited swap
+      --memory-swappiness int       Tune container memory swappiness (0 to 100) (default -1).
+      --name string                 Assign a name to the container
+      --net string                  Connect a container to a network (default "default")
                                     'bridge': create a network stack on the default Docker bridge
                                     'none': no networking
                                     'container:<name|id>': reuse another container's network stack
                                     'host': use the Docker host network stack
                                     '<network-name>|<network-id>': connect to a user-defined network
-      --net-alias=[]                Add network-scoped alias for the container
-      --oom-kill-disable            Whether to disable OOM Killer for the container or not
-      --oom-score-adj=0             Tune the host's OOM preferences for containers (accepts -1000 to 1000)
-      -P, --publish-all             Publish all exposed ports to random ports
-      -p, --publish=[]              Publish a container's port(s) to the host
-      --pid=""                      PID namespace to use
-      --pids-limit=-1                Tune container pids limit (set -1 for unlimited), kernel >= 4.3
+      --net-alias value             Add network-scoped alias for the container (default [])
+      --no-healthcheck              Disable any container-specified HEALTHCHECK
+      --oom-kill-disable            Disable OOM Killer
+      --oom-score-adj int           Tune host's OOM preferences (-1000 to 1000)
+      --pid string                  PID namespace to use
+      --pids-limit int              Tune container pids limit (set -1 for unlimited)
       --privileged                  Give extended privileges to this container
+  -p, --publish value               Publish a container's port(s) to the host (default [])
+  -P, --publish-all                 Publish all exposed ports to random ports
       --read-only                   Mount the container's root filesystem as read only
-      --restart="no"                Restart policy (no, on-failure[:max-retry], always, unless-stopped)
+      --restart string              Restart policy to apply when a container exits (default "no")
```

Comment:
Perhaps we should keep the available options here (ideally in the help output as well, but there's more places where we have that issue)